tools/clearly-defined/check-clearly-defined.ps1

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+Determine if this is a dependabot PR. If it is and the package being updated is not excluded, check
+if ClearlyDefined has information for the current version. Fail the build is it's not there.
+
+.DESCRIPTION
+ClearlyDefined is our source of license information for dependemncies, but it often lags behind
+the latest version of a package. This script exists to detect this scenario and to serve as a
+reminder to request that ClearlyDefined harvest information for the version being updated. It
+also provides a way to exclude packages that we know will never be in ClearlyDefined.
+
+This script will be in multiple repos, please try to keep it in sync.
```

Comment:
When and if we copy this to multiple places, can we maintain a list (perhaps here) of those repos to help future updaters?

Reply:
Added